Zend/tests/type_declarations/mixed/validation/mixed_property_weak_success.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
```

Comment:
Isn't this test duplicated from `Zend/tests/type_declarations/mixed/validation/mixed_property_success.phpt`?

Reply:
Yes, it was, thanks for pointing it out!